Library/Homebrew/language/python.rb

```diff
@@ -112,6 +113,24 @@ def detected_python_shebang(formula = self)
 
     # Mixin module for {Formula} adding virtualenv support features.
     module Virtualenv
+      PYTHON_VIRTUALENV_URL =
+        "https://files.pythonhosted.org/packages/11/74" \
+        "/2c151a13ef41ab9fb43b3c4ff9e788e0496ed7923b2078d42cab30622bdf" \
+        "/virtualenv-16.7.4.tar.gz"
+      private_constant :PYTHON_VIRTUALENV_URL
+      PYTHON_VIRTUALENV_SHA256 =
+        "94a6898293d07f84a98add34c4df900f8ec64a570292279f6d91c781d37fd305"
+      private_constant :PYTHON_VIRTUALENV_SHA256
+
+      PYTHON_VIRTUALENV_URL_MOJAVE =
+        "https://files.pythonhosted.org/packages/b1/72" \
+        "/2d70c5a1de409ceb3a27ff2ec007ecdd5cc52239e7c74990e32af57affe9" \
+        "/virtualenv-15.2.0.tar.gz"
+      private_constant :PYTHON_VIRTUALENV_URL_MOJAVE
+      PYTHON_VIRTUALENV_SHA256_MOJAVE =
+        "1d7e241b431e7afce47e77f8843a276f652699d1fa4f93b9d8ce0076fd7b0b54"
+      private_constant :PYTHON_VIRTUALENV_SHA256_MOJAVE
+
```

Comment:
1. Is there a reason why this is not
    ```rb
    if MacOS.version
      PYTHON_VIRTUALENV_URL = ...
    ...
    else
      PYTHON_VIRTUALENV_URL = ...
    ...
    ```

2. Below we use `MacOS.version` but there is no `require "os/mac/version"` or `require "os/mac"`. Are we sure we don't need it on a Mac?

3. On Linux `MacOS.version` defaults to `NULL`, so Linux defaults to using `PYTHON_VIRTUALENV_URL_MOJAVE` -- I think it's better to make it default to using virtualenv-16.7.4.

Reply:
I'm not really sure how this should behave.

Reply:
`PYTHON_VIRTUALENV_URL_MOJAVE` purely exists for system Python 2, which I don't think we really support anymore.

Reply:
Can try to kill it here.